Gemfile

```diff
@@ -92,3 +92,8 @@ gem 'axlsx', '= 2.0.1'
 gem 'axlsx_rails'
 
 gem 'roo'
+
+gem 'responders'
+gem 'react-rails'
```

Comment:
Experimenting with react views in rails but not implemented client side views in react yet.